utils/_context/_scenarios/parametric.py

```diff
@@ -490,6 +487,10 @@ def dotnet_library_factory():
 # "disable" rate limiting by default by setting it to a large value
 ENV DD_TRACE_RATE_LIMIT=10000000
 
+COPY --from=build /app/out /app
+COPY --from=build /app/SYSTEM_TESTS_LIBRARY_VERSION /app/SYSTEM_TESTS_LIBRARY_VERSION
+COPY --from=build /opt/datadog /opt/datadog
```

Comment:
Any particular reason to move this down below the `ENV` lines?

Reply:
Same as above, being as close as possible to the weblog def 😆